runtime/pubsub/internal/gcp/clients.go

```diff
@@ -10,7 +10,7 @@ import (
 func (mgr *Manager) getClient() *pubsub.Client {
 	mgr.clientOnce.Do(func() {
 		// Create a new client
-		cl, err := pubsub.NewClient(mgr.ctxs.Connection, "")
+		cl, err := pubsub.NewClient(mgr.ctxs.Connection, "-")
```

Comment:
I would have updated our runtime version as well here, rather than just do this as the newer versions have security fixes we should include

Reply:
Agreed, but seemed reasonable to do that as a separate change